spec/models/form526_submission_spec.rb

```diff
@@ -98,72 +98,87 @@
+    let!(:happy_path_success) { create(:form526_submission, :with_submitted_claim_id) }
+    let!(:pending_backup) { create(:form526_submission, :backup_path) }
+    let!(:accepted_backup) { create(:form526_submission, :backup_path, :backup_accepted) }
+    let!(:rejected_backup) { create(:form526_submission, :backup_path, :backup_rejected) }
+    let!(:remediated) { create(:form526_submission, :remediated) }
+    let!(:remediated_and_expired) { create(:form526_submission, :remediated) }
+    let!(:remediated_and_rejected) { create(:form526_submission, :remediated, :backup_path) }
+    let!(:no_longer_remediated) { create(:form526_submission, :no_longer_remediated) }
 
+    describe 'pending_backup_submissions' do
       it 'returns records submitted to the backup path but lacking a decisive state' do
         expect(Form526Submission.pending_backup_submissions).to contain_exactly(
-          backup_submission_a,
-          backup_submission_b
+          pending_backup
         )
       end
     end
 
     describe 'in_process' do
-      let!(:in_process_submission1) { create(:form526_submission) }
-      let!(:in_process_submission2) { create(:form526_submission, :backup_path) }
-      let!(:expired_submission) { create(:form526_submission, :backup_path, :created_more_than_3_days_ago) }
-      let!(:backup_accepted_submission) { create(:form526_submission, :backup_path, :with_accepted_backup_status) }
-      let!(:successful_submission) { create(:form526_submission, :with_submitted_claim_id) }
-
-      it 'only returns submissions that are in process' do
-        result = Form526Submission.in_process
-
-        expect(result).to include(in_process_submission1, in_process_submission2)
-        expect(result).not_to include(expired_submission, backup_accepted_submission, successful_submission)
+      it 'only returns submissions that are still in process' do
+        expect(Form526Submission.in_process).to contain_exactly(
+          in_process,
+          pending_backup
+        )
       end
     end
 
-    describe 'success_type' do
-      let!(:successful_submission) { create(:form526_submission, :with_submitted_claim_id) }
-      let!(:backup_accepted_submission) { create(:form526_submission, :backup_path, :with_accepted_backup_status) }
-      let!(:in_process_submission) { create(:form526_submission) }
-      let!(:remediated_submission) do
-        create(:form526_submission_remediation, form526_submission: subject)
-        subject
+    describe 'accepted_to_primary_path' do
+      it 'returns submissions with a submitted_claim_id' do
+        expect(Form526Submission.accepted_to_primary_path).to contain_exactly(
+          happy_path_success
+        )
       end
+    end
 
-      it 'only returns submissions that are successful types' do
-        result = Form526Submission.success_type
+    describe 'accepted_to_backup_path' do
+      it 'returns submissions with a backup_submitted_claim_id that have been explicitly rejected' do
```

Comment:
Should this be submissions that have been explicitly **accepted**, not **rejected**?

Reply:
yes good catch